modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,135 @@
+= Footnotes plugin
+:navtitle: Footnotes
+:description: Insert footnotes into the {productname} editor
+:keywords: footnotes, footnote, insertfootnotes, insert, footnotesupdate, mceInsertFootnote, mceUpdateFootnotes
+:pluginname: Footnotes
+:plugincode: footnotes
+:pluginminimumplan: tierthree
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a footnote into the {productname} editor.
+
+Each time a footnote is added, a superscripted footnote number is placed at the current insertion point, or at the end of the current selection.
+
+The plugin then adds a new line to the footnotes section located at the bottom of the editor and moves the insertion point to this new line. This allows for the immediate creation of the contents of the footnote entry related to the newly-created footnote
+
+== Interactive Example
+
+liveDemo::footnotes[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'footnotes',
+  toolbar: 'footnotes',
+});
+----
+
+The footnotes section, which is placed at the bottom of the {productname} editor, is created when the first footnote is added.
+
+Footnotes can be inserted using the _Insert footnote_ button, located on the toolbar, or by selecting the _Insert > Insert Footnote_ command from the menu bar, or by using the `+mceInsertFootnote+` command.
```

Comment:
There's 2 or's here which reads rather strangely. Is that right or should that be reworded to avoid the heavy if/else part of this sentence?

Reply:
Re-set how-to-insert-a-footnote as a bulleted list.

Also corrected block comment marker to line comment marker (and internally grouched, yet again, at AsciiDoc having two different ways of demarcating comments for no good reason I can think of).